contracts/src/v0.8/automation/dev/v2_2/AutomationRegistry2_2.sol

```diff
@@ -327,6 +327,7 @@ contract AutomationRegistry2_2 is AutomationRegistryBase2_2, OCR2Abstract, Chain
     });
     s_fallbackGasPrice = onchainConfig.fallbackGasPrice;
     s_fallbackLinkPrice = onchainConfig.fallbackLinkPrice;
+    skipReorgProtection = onchainConfig.skipReorgProtection;
```

Comment:
n00b: why prefix s_? with prefix s_, s_fallbackLinkPrice is a state variable (persistent storage onchain)? skipReorgProtection is just a local variable which will not be recorded onchain?

Reply:
s_ means this is a storage variable
i_ means this is a immutable variable